src/app/dashboard/user-studies.tsx

```diff
@@ -1,43 +1,54 @@
 'use client'
 
-import { UserName } from '@/components/user-name'
+import { ResearcherUserStudiesTable } from '@/components/dashboard/researcher-user-studies-table'
+import { ReviewerUserStudiesTable } from '@/components/dashboard/reviewer-user-studies-table'
 import { useSession } from '@/hooks/session'
+import { getEnclaveOrg, getLabOrg } from '@/lib/types'
 import { Flex, Paper, SegmentedControl, Stack, Text, Title } from '@mantine/core'
 import { useState } from 'react'
-import { ResearcherUserStudiesTable } from '@/components/dashboard/researcher-user-studies-table'
-import { ReviewerUserStudiesTable } from '@/components/dashboard/reviewer-user-studies-table'
 
 export default function UserStudiesDashboard() {
     const { session } = useSession()
     const [activeTab, setActiveTab] = useState<'researcher' | 'reviewer'>('reviewer')
 
     if (!session) return null
 
+    // Determine the org types the user belongs to
+    const hasLabOrg = Boolean(getLabOrg(session))
+    const hasDataOrg = Boolean(getEnclaveOrg(session))
```

Comment:
I just added these to the session, it's on session.memberOfLab and session.memberOfEnclave